dist/info/mame2003-plus-libretro.info

```diff
@@ -0,0 +1,13 @@
+display_name = "Arcade (MAME 2003-Plus - Experimental)"
+authors = "MAMEdev"
+supported_extensions = "zip"
+corename = "MAME 2003 Plus"
```

Comment:
That's a question I'd like to get your opinion on.

MAME 2003+ is a working title and there is discussion of taking the 2003 out of the name. (I personally like LightningMAME, since the old LM is dead).

There is more code from MAME 0.78 than any other MAME but but I believe mame2003-plus currently it has code from MAMEs dating 2001-2009 at this point.

For example, the needed romsets are a bit different than the MAME 0.78 although if you have a MAME 0.78 collection you are probably going to have at least as good a compatibility as with mame2003/mame 0.78.

Anyway maybe this is all academic until the core actually gets renamed. I'd like to go with whatever you think is appropriate/consistent.